task/buildah/0.1/buildah.yaml

```
@@ -502,17 +502,22 @@ spec:
         echo "WARNING: provided deprecated BUILDER_IMAGE parameter has no effect."
       fi
 
+      echo "Inspecting $IMAGE to find its base image name and digest"
       base_image_name=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.name"}}' $IMAGE | cut -f1 -d'@')
       base_image_digest=$(buildah inspect --format '{{ index .ImageAnnotations "org.opencontainers.image.base.digest"}}' $IMAGE)
+
       container=$(buildah from --pull-never $IMAGE)
+      echo "Injecting sbom files into /root/buildinfo/content_manifests/ of $IMAGE"
       buildah copy $container sbom-cyclonedx.json sbom-purl.json /root/buildinfo/content_manifests/
+      echo "Re-applying base image annotations and labels"
```

```suggestion
      echo "Re-applying base image annotations"
```
Here as well, if we're keeping the `0.1` changes

---

Doh, how did I miss that. Thanks!

---

Regarding old task versions, I'd like to add it anyways since - it's for me. I need the visual aid when reviewing the output of buildah tasks and trying to confirm that manifests are being injected where they should be.

(Long story, but the ai-lab images are so huge that ec times out trying to verify that the files exist in /root/buildinfo/, but it appears to suppress the timeout and instead just reports that the files are missing. Related to https://github.com/enterprise-contract/ec-cli/issues/1788)